acceptance/cmd/workspace/query-history/test.toml

```diff
@@ -0,0 +1,71 @@
+Local = true
+Cloud = false
+
+[[Repls]]
+Old="Date: .*"
+New="Date: (redacted)"
+
+[[Repls]]
+Old="Traceparent: .*"
+New="Traceparent: (redacted)"
```

Comment:
I think I've seen this Repls in other PRs. If you anticipate them being generally useful, it would be good idea to move them to cmd/test.toml or cmd/workspace/test.toml